webapp/resources/cas-theme-default.properties

```diff
@@ -1,3 +1,3 @@
-standard.custom.css.file=/css/cas.css
-admin.custom.css.file=/css/admin.css
+cas.standard.css.file=/css/cas.css
```

Comment:
Documentation must be updated with the new settings.

Reply:
Will update the documentation